types/fs-extra/test/fs-extra-tests-module.ts

```diff
@@ -0,0 +1,299 @@
+import * as fs from 'fs-extra/esm.mjs';
+
+// test type exports
+type CopyFilterSync = fs.CopyFilterSync;
+type CopyFilterAsync = fs.CopyFilterAsync;
+type SymlinkType = fs.SymlinkType;
+type CopyOptions = fs.CopyOptions;
+type CopyOptionsSync = fs.CopyOptionsSync;
+type EnsureOptions = fs.EnsureOptions;
+type MoveOptions = fs.MoveOptions;
+type ReadOptions = fs.ReadOptions;
+type WriteFileOptions = fs.WriteFileOptions;
+type WriteOptions = fs.WriteOptions;
```

Comment:
Can we test the same (test types) with an import expected to be used in ESM?
```suggestion
import fs, {
    CopyFilterAsync,
    CopyFilterSync,
    CopyOptions,
    CopyOptionsSync,
    EnsureOptions,
    MoveOptions,
    ReadOptions,
    SymlinkType,
    WriteFileOptions,
    WriteOptions,
} from 'fs-extra/esm.mjs';

// test type exports
type Types =
    | CopyFilterSync
    | CopyFilterAsync
    | CopyOptions
    | CopyOptionsSync
    | EnsureOptions
    | MoveOptions
    | ReadOptions
    | SymlinkType
    | WriteFileOptions
    | WriteOptions;
```

Reply:
done